Resources/config/services.yml

```diff
@@ -0,0 +1,24 @@
+services:
+    # default configuration for services in *this* file
+    _defaults:
+        # automatically injects dependencies in your services
+        autowire: true
+        # automatically registers your services as commands, event subscribers, etc.
+        autoconfigure: true
+        # this means you cannot fetch services directly from the container via $container->get()
+        # if you need to do this, you can override this setting on individual services
+        public: false
+
+    # controllers are imported separately to make sure they're public
+    # and have a tag that allows actions to type-hint services
+    MovingImage\Bundle\MICommentsBundle\Controller\:
+        resource: '../../Controller'
+        public: true
+        tags: ['controller.service_arguments']
```

Comment:
Maybe this line is not necessary, since autoconfigure is set to true and the controllers extend from the Symfony Controller

Reply:
I think you are right.